modules/gis/src/lib/binary-to-geojson.ts

```diff
@@ -90,7 +92,11 @@ function parseFeatures(data, type): Geometry | Feature[] {
 }
 
 /** Parse input binary data and return a valid GeoJSON geometry object */
-function parseGeometry(data: BinaryGeometry, startIndex?: number, endIndex?: number): Geometry {
+export function parseGeometry(
```

Comment:
Hmm I am getting confused. Aren't we already exporting `binaryToGeometry`? What does this do differently?

Reply:
This is the function that is exported as `binaryToGeometry`.

Reply:
I think this is exported as `binaryToGeometry`:

```js
export {binaryToGeoJSON, parseGeometry as binaryToGeometry} from './lib/binary-to-geojson';
```

Maybe simpler to just name this function `binaryToGeometry`.

Reply:
+1

Reply:
Renamed to `binaryToGeometry`